src/longopt/gmtspatial_inc.h

```diff
@@ -25,53 +25,44 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 		  short_directives,    long_directives,
 		  short_modifiers,     long_modifiers,
 		  transproc_mask */
-	{ 0, 'A', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'C', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'D', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'E', "",
-	          "",                  "",
+	{ 0, 'A', "nn|nearest_neighbor",
+	          "a",                 "min_dist",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'F', "",
-	          "",                  "",
+	{ 0, 'C', "clip",              "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'D', "duplicates",
 	          "",                  "",
+	          "a,c,C,d,f,p,s",     "amax,cmax,cmin,dmax,file,perpendicular,factor",
 		  GMT_TP_STANDARD },
-	{ 0, 'I', "",
-	          "",                  "",
+	{ 0, 'E', "handedness",
 	          "",                  "",
+	          "p,n",               "positive|counterclockwise,negative|clockwise",
 		  GMT_TP_STANDARD },
-	{ 0, 'L', "",
-	          "",                  "",
+	{ 0, 'F', "force_polygons",
+	          "l",                 "lines",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'N', "",
-	          "",                  "",
+	{ 0, 'I', "intersections",
+	          "e,i",               "external,internal",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'Q', "",
-	          "",                  "",
+	{ 0, 'L', "no_tile_lines",     "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'N', "in_polygons",
 	          "",                  "",
+	          "a,i,p,r,z",         "all,individual,start,report,addcolumn",
 		  GMT_TP_STANDARD },
-	{ 0, 'S', "",
-	          "",                  "",
+	{ 0, 'Q', "centroid|area|length",
 	          "",                  "",
+	          "c,h,l,p,s",         "range|limits,header,lines,close,sort",
 		  GMT_TP_STANDARD },
-	{ 0, 'T', "",
-	          "",                  "",
+	{ 0, 'S', "spatial",
+	          "b,h,i,j,s,u",       "buffer,hole,intersection,join,split,union",
```

Comment:
The directives (some) are also different in Julia. GMT.jl has `holes` instead of `hole` and `dateline` instead of `split`. I think `dateline` is better because it clearly indicates where the split occurs.

Reply:
Tend to agree with this. Could do alias holes here but I like dateline.

Reply:
We can add alias holes but not that all the others are singular nouns so _holes_ stand out.

Reply:
Best if GMT.jl now adds the hole alias to match CLI.  I did the dateline change and added alias holes.